common.go

```diff
@@ -154,3 +155,18 @@ func secureRandomBytes(bytes uint) ([]byte, error) {
 	_, err := rand.Read(r)
 	return r, err
 }
+
+// secureCompare will compare two slice of bytes in constant time, ensuring no timing information
+// is leaked in order to prevent timing attacks.
+func secureCompare(a, b []byte) bool {
+	if len(a) != len(b) {
```

Comment:
@benmanns I don't think this is even being used but it's also the same implementation that crypto/subtle uses, I wasn't aware of that method in the stdlib.